recipes/mp-units/2.2.0/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
feel free to remove the `test_v1_package`, no longer mandatory for new recipes (I'm considering 2.2.0 a new recipe here :) )

Reply:
Great!